Felienne/hedy
# [UI FIX] Re-directs /explore to /login when not logged in

**Description**
Currently we throw users a "not authorized" error when trying to visit the explore page without being logged in. This works fine, but it suggest that for some reason they are not allowed at all to visit that page. This is incorrect as they can visit the page as soon as they are logged in. In this PR we redirect the user to the /login page instead of throwing an error.

**Fix for**
This PR fixes #1977.

**How to test**
Make sure you are not logged in and attempt to visit the /explore page. Verify that you are correctly re-directed to the login page.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
